src/smc-webapp/projects.cjsx

```diff
@@ -46,6 +46,17 @@ MAX_DEFAULT_PROJECTS = 50
 
 _create_project_tokens = {}
 
+window.restore_tabs = () ->
```

Comment:
It would be better to make these two methods on `window.smc`, since `window.[blah]` could conflict easily.